src/main/java/com/crystal/Monster.java

```diff
@@ -26,4 +27,55 @@ public void rankMonster() {
             System.out.println("all-seeing behemoth");
         }
     }
+
+
+    @Override
+    public String getName() {
+        return this.name;
+    }
+
+    @Override
+    public String getClassName() {
+        return "Monster";
+    }
+
+    @Override
+    public boolean isScared() {
+        return scared;
+    }
+
+    @Override
+    public void setScared(boolean scared) {
+        this.scared = scared;
+    }
+
+    @Override
+    public void isInjured(String attack) {
+
+    }
+
+    @Override
+    public boolean isDead() {
+        return false;
+    }
+
+    @Override
+    public boolean kick(Character characterBeingAttacked) {
+        return false;
+    }
+
+    @Override
+    public boolean punch(Character characterBeingAttacked) {
+        return false;
+    }
+
+    @Override
+    public boolean headbutt(Character characterBeingAttacked) {
+        return false;
+    }
+
+    @Override
+    public boolean roar(Character characterBeingAttacked) {
+        return false;
+    }
```

Comment:
Can we make these private and have the attack method call them?

Reply:
done